starboard/shared/modular/starboard_layer_posix_fcntl_abi_wrappers.cc

```diff
@@ -0,0 +1,156 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard/shared/modular/starboard_layer_posix_fcntl_abi_wrappers.h"
+
+#include <fcntl.h>
+#include <stdarg.h>
+
+#include "starboard/common/log.h"
+
+int MuslCmdToPlatformCmd(int musl_cmd) {
+  switch (musl_cmd) {
```

Comment:
I think we still need F_DUPFD_CLOFORK, F_GETOWN_EX, F_SETOWN_EX, F_OFD_GETLK, F_OFD_SETLK, F_OFD_GETLKW